kani-compiler/src/codegen_cprover_gotoc/codegen/rvalue.rs

```diff
@@ -472,19 +472,25 @@ impl<'tcx> GotocCtx<'tcx> {
                         FieldsShape::Arbitrary { offsets, .. } => offsets[0].bytes_usize(),
                         _ => unreachable!("niche encoding must have arbitrary fields"),
                     };
-                    let discr_ty = self.codegen_enum_discr_typ(ty);
-                    let discr_ty = self.codegen_ty(discr_ty);
-                    let niche_val = self.codegen_get_niche(e, offset, discr_ty.clone());
-                    let relative_discr = if *niche_start == 0 {
-                        niche_val
-                    } else {
-                        wrapping_sub(&niche_val, *niche_start)
-                    };
+
+                    // Compute relative discriminant value (`niche_val - niche_start`).
```

Comment:
Is there Rust documentation to point to here?

Reply:
Nope... I can point to the code where I derived this comment from though.